templates/org/projects/view.tmpl

```diff
@@ -8,8 +8,8 @@
 			{{template "user/overview/header" .}}
 		</div>
 	{{end}}
-	<div class="ui container fluid padded">
-		{{template "projects/view" .}}
+	<div class="ui container fluid padded org-projects-view">
+		{{template "projects/view" dict "." . "FullscreenElementsSelector" ".org-projects-view"}}
```

Comment:
The old layout is not right, the new change makes things worse.

`projects/view` already has its own "ui container" and it could have a unified CSS class name.

Reply:
I'm very sorry. From what I can see on the page, I don't find anything inappropriate. I can't tell what's wrong with the layout. Or rather, I don't understand what you think the correct layout should be.

Reply:
See latest commits